phpstan.neon

```diff
@@ -0,0 +1,7 @@
+parameters:
+  level: 9
+  paths:
+    - src/
+    - views/
+    - wp-parsely.php
+    - uninstall.php
```

Comment:
```suggestion
    - uninstall.php
  inferPrivatePropertyTypeFromConstructor: true
  ignoreErrors:
    # Uses func_get_args()
    - '#^Function apply_filters(_ref_array)? invoked with [34567] parameters, 2 required\.$#'
```
The new property is documented at https://phpstan.org/config-reference#inferprivatepropertytypefromconstructor

This ignore removes 11 Level 1 issues that we wouldn't be able to otherwise address.